users/views.py

```diff
@@ -46,4 +46,45 @@ def get(self, request):
         
         except SocialLogin.DoesNotExist:
             return JsonResponse({'message': 'INVALID_USER'}, status=404)
-        
\ No newline at end of file
+
+class UserDataView(View):
```

Comment:
유저의 프로필 관련 로직인 것 같으니 네이밍을 좀 더 명확하게 해주시면 좋을 것 같아요!
```suggestion
class ProfileView(View):
```